src/registrar/models/domain_application.py

```diff
@@ -28,62 +28,69 @@ class DomainApplication(TimeStampedModel):
     ]
 
     class StateTerritoryChoices(models.TextChoices):
-        ALABAMA = "AL", "Alabama"
-        ALASKA = "AK", "Alaska"
-        ARIZONA = "AZ", "Arizona"
-        ARKANSAS = "AR", "Arkansas"
-        CALIFORNIA = "CA", "California"
-        COLORADO = "CO", "Colorado"
-        CONNECTICUT = "CT", "Connecticut"
-        DELAWARE = "DE", "Delaware"
-        DISTRICT_OF_COLUMBIA = "DC", "District of Columbia"
-        FLORIDA = "FL", "Florida"
-        GEORGIA = "GA", "Georgia"
-        HAWAII = "HI", "Hawaii"
-        IDAHO = "ID", "Idaho"
-        ILLINOIS = "IL", "Illinois"
-        INDIANA = "IN", "Indiana"
-        IOWA = "IA", "Iowa"
-        KANSAS = "KS", "Kansas"
-        KENTUCKY = "KY", "Kentucky"
-        LOUISIANA = "LA", "Louisiana"
-        MAINE = "ME", "Maine"
-        MARYLAND = "MD", "Maryland"
-        MASSACHUSETTS = "MA", "Massachusetts"
-        MICHIGAN = "MI", "Michigan"
-        MINNESOTA = "MN", "Minnesota"
-        MISSISSIPPI = "MS", "Mississippi"
-        MISSOURI = "MO", "Missouri"
-        MONTANA = "MT", "Montana"
-        NEBRASKA = "NE", "Nebraska"
-        NEVADA = "NV", "Nevada"
-        NEW_HAMPSHIRE = "NH", "New Hampshire"
-        NEW_JERSEY = "NJ", "New Jersey"
-        NEW_MEXICO = "NM", "New Mexico"
-        NEW_YORK = "NY", "New York"
-        NORTH_CAROLINA = "NC", "North Carolina"
-        NORTH_DAKOTA = "ND", "North Dakota"
-        OHIO = "OH", "Ohio"
-        OKLAHOMA = "OK", "Oklahoma"
-        OREGON = "OR", "Oregon"
-        PENNSYLVANIA = "PA", "Pennsylvania"
-        RHODE_ISLAND = "RI", "Rhode Island"
-        SOUTH_CAROLINA = "SC", "South Carolina"
-        SOUTH_DAKOTA = "SD", "South Dakota"
-        TENNESSEE = "TN", "Tennessee"
-        TEXAS = "TX", "Texas"
-        UTAH = "UT", "Utah"
-        VERMONT = "VT", "Vermont"
-        VIRGINIA = "VA", "Virginia"
-        WASHINGTON = "WA", "Washington"
-        WEST_VIRGINIA = "WV", "West Virginia"
-        WISCONSIN = "WI", "Wisconsin"
-        WYOMING = "WY", "Wyoming"
-        AMERICAN_SAMOA = "AS", "American Samoa"
-        GUAM = "GU", "Guam"
-        NORTHERN_MARIANA_ISLANDS = "MP", "Northern Mariana Islands"
-        PUERTO_RICO = "PR", "Puerto Rico"
-        VIRGIN_ISLANDS = "VI", "Virgin Islands"
+        ALABAMA = "AL", "Alabama (AL)"
```

Comment:
**comment:** This is great to see for keyboard accessibility of the states select box, thank you.